src/annotator/test/sidebar-test.js

```diff
@@ -760,6 +762,21 @@ describe('Sidebar', () => {
           width: 0,
         });
       });
+
+      it('removes the iframe from the container when destroyed', () => {
+        sidebar.show();
+        assert.exists(sidebar.iframe.parentElement);
+        sidebar.destroy();
+        assert.notExists(sidebar.iframe.parentElement);
+      });
+
+      it('ignores pan events', () => {
```

Comment:
This test seems unrelated to the other changes in this PR. Can you clarify why it was added?

Reply:
A new addition is that now we remove `this.iframe` from the `externalContainer` on `destroy`. That's was something overlooked previously. That's the reason for the extra test.